docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+
+Do the following to generate the file:
+
+1. Pick a Node to reconfigure. We will refer to this Node's name as NODE_NAME.
+2. Start the kubectl proxy in the background with `kubectl proxy --port=8001 &`
+3. Run the following command to download and unpack the configuration from the configz endpoint: 
+
+```
+curl http://localhost:8001/api/v1/proxy/Nodes/NODE_NAME/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubeletconfig/v1alpha1"' > kubelet
+```
+
+Note that we have to manually add the `kind` and `apiVersion` to the downloaded object, as these are not reported by the configz endpoint. This is one of the limitations of the endpoint that is planned to be fixed in the future.
+
+Note that the configuration must be in a file named `kubelet`, so that the data appears under the `kubelet` key of the ConfigMap.
+
+### Edit the configuration file
+
+Using your editor of choice, change one of the parameters in the `kubelet` file from the previous step. A QPS parameter, `eventRecordQPS` for example, is a good candidate.
+
+### Push the configuration file to the control plane
+
+Push the edited configuration file to the control plane with the following command:
+
+```
+kubectl create configmap my-Node-config --namespace=kube-system --from-file kubelet --append-hash -o yaml
+```
+
+You should see a response similar to:
+
+```
+apiVersion: v1
+data:
+  kubelet: |
+    {...}
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2017-09-14T20:23:33Z
+  name: my-Node-config-gkt4c2m4b2
+  namespace: kube-system
+  resourceVersion: "119980"
+  selfLink: /api/v1/namespaces/kube-system/configmaps/my-Node-config-gkt4c2m4b2
+  uid: 946d785e-998a-11e7-a8dd-42010a800006
+```
+
+We create the ConfigMap in the `kube-system` namespace, which is appropriate because this ConfigMap configures a Kubernetes system component - the Kubelet.
+
+The `--append-hash` option appends a short checksum of the ConfigMap contents to the name. This is convenient for an edit->push workflow, as it will automatically, yet deterministically, generate new names for new ConfigMaps. 
+
+We use the `-o yaml` output format so that the name, namespace, and uid are all reported following creation. We will need these in the next step. We will refer to the name as CONFIG_MAP_NAME and the uid as CONFIG_MAP_UID.
+
+### Set the Node to use the new configuration
+
+Edit the Node's reference to point to the new ConfigMap with the following command:
+
+`kubectl edit Node NODE_NAME`
+
+Once in your editor, add the following under `spec`:
```

Comment:
can you please also add an example of the patch command for users that aren't familar with that?